packages/wdio-sync/src/wrapCommand.js

```diff
@@ -118,6 +118,17 @@ function isNotInFiber (context, fnName) {
  * @param {object} context browser or element
  */
 function inFiber(context) {
+    if (context.constructor.name === 'MultiRemoteDriver') {
```

Comment:
I think this one might not work if we are in element context.
Ex: 
browser // MultiRemoteDriver
browser.$('foo') // ?

Reply:
now I know that it can be also `Element` and `Browser`